web/src/copy_and_paste.js

```diff
@@ -341,7 +341,28 @@ export function paste_handler(event) {
     }
 
     if (clipboardData.getData) {
+        const $textarea = $(event.currentTarget);
+        const paste_text = clipboardData.getData("text");
         const paste_html = clipboardData.getData("text/html");
+        // Trim the paste_text to accommodate sloppy copying
+        const trimmed_paste_text = paste_text.trim();
+
+        // Match for anything that resembles a URL, something://something
+        const url_regex = /^\w+:\/\/\S+$/;
```

Comment:
Surely we have a better "is this probably a URL" check we can do, either via native libraries or already in the codebase; @andersk do you have something to suggest?

Reply:
Good point. I didn't see anything in the frontend code that looked like it was checking for URLs, so let me push an update using the [is-url](https://github.com/segmentio/is-url) package, which looks pretty good and has a nice set of tests.

I'm still interested in Anders's opinion, too.

Reply:
Yeah that seems fine for now; we can always refactor from here if it seems warranted.